bin/web/index.js

```diff
@@ -9,14 +9,14 @@ const start = (port) => {
   dalai.http(httpServer)
   app.use(express.static(path.resolve(__dirname, 'public')))
   app.use(express.json());
-  app.use(express.urlencoded());
+  app.use(express.urlencoded({ extended: true }));
```

Comment:
Good.  Fixes bug where server emits this line on stating:  

body-parser deprecated undefined extended: provide extended option .npm/_npx/3c737cbb02d79cc9/node_modules/dalai/bin/web/index.js:12:19